config/samples/install_v1alpha1_lookout.yaml

```diff
@@ -21,8 +20,8 @@ spec:
   hostNames: [localhost]
   replicas: 2
   image:
-    repository: gresearch/armada-lookout
-    tag: 6141171b8bde8a03113090a819d728511c2dc39f
+    repository: gresearchdev/armada-lookout
```

Comment:
I’m not sure this change is correct. @richscott told me that we migrated our repos to use gresearch not gresearchdev.  I don’t know where those releases come from  anymore

Reply:
insofar as this is a sample I thought it would be more exemplary to use a tagged release? this now matches the other samples as well